scripts/update_changelog_and_credits_test.py

```diff
@@ -99,6 +100,12 @@ def mock_run_cmd(unused_cmd):
         def mock_get_git_ref(unused_self, unused_ref):
             return github.GitRef.GitRef(
                 requester='', headers='', attributes={}, completed='')
+        def mock_main(unused_personal_access_token):
+            pass
+        # pylint: disable=unused-argument
+        def mock_getpass(prompt):
```

Comment:
Can't you do unused_prompt and avoid the pragma?

Reply:
`prompt` is passed as a named argument in the `getpass` function, so we cannot change it to `unused_prompt`.